src/component/TitleBar.tsx

```
@@ -30,34 +29,53 @@ export default function TitleBar() {
             <div className="pointer-events-none select-none">
                 <h1>{title}</h1>
             </div>
-            <div className="flex select-none h-full justify-center items-center">
-                <div onClick={minHandler} className="hover:bg-bg-secondary h-full flex items-center px-2 transition-colors">
-                    <img src={minimizeButton} className="pointer-events-none" alt="minimize" />
+            <div className="flex h-full select-none items-center justify-center">
+                <div
+                    onClick={minHandler}
+                    onKeyDown={(e) => e.key === "Enter" && minHandler()}
+                    tabIndex={0}
+                    role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)